developer/ruby/GenerateClass.rb

```diff
@@ -300,21 +330,21 @@ def forwardTranslate(options)
 
 # START CLASSES
 
-hpp << "class " << sourceFolders[0].upcase << "_API " << className
+hpp << "  class " << sourceFolders[0].upcase << "_API " << className
```

Comment:
Lots of whitespace changes because I wanted it to be formatted the closest to what clang-format would end up doing to begin with. In case the user doesn't have clang-format in the PATH, this will minimize later useless diffs